src/App.js

```diff
@@ -2,26 +2,33 @@ import React from 'react';
 import CreateItem from './components/CreateItem';
 import ShowList from './components/ShowList';
 import Home from './components/Home';
-import Navigation from './components/Navigation';
+
 import './App.css';
-import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import {
+  Redirect,
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
 
 const App = () => {
+  const existingToken = localStorage.getItem('token');
+
   return (
     <div className="App">
       <Router>
         <Switch>
+          <Route exact path="/">
+            <Home />
+          </Route>
           <Route path="/list-view">
             <ShowList />
           </Route>
           <Route path="/add-view">
             <CreateItem />
           </Route>
-          <Route path="/">
-            <Home />
-          </Route>
         </Switch>
-        <Navigation />
+        {existingToken ? <Redirect to="/list-view" /> : <Redirect to="/" />}
```

Comment:
This looks great! Nice use of the ternary operator here :)

Reply:
I agree. Great use of ternary 🚀